lib/glific/contacts/import.ex

```diff
@@ -32,14 +32,25 @@ defmodule Glific.Contacts.Import do
       language_id: Enum.at(Settings.get_language_by_label_or_locale(data["language"]), 0).id,
       collection: data["collection"],
       delete: data["delete"],
-      contact_fields: Map.drop(data, ["phone", "group", "language", "delete", "opt_in"]),
-      optin_time:
-        if(data["opt_in"] in ["", nil],
-          do: nil,
-          else: elem(Timex.parse(data["opt_in"], date_format), 1)
-        )
+      contact_fields: Map.drop(data, ["phone", "group", "language", "delete", "opt_in"])
     }
 
+    results =
```

Comment:
After getting few errors from credo time to time I started relying on if block more. 

```elixir
    if data["opt_in"]  in ["", nil], do: results, else: Map.put(results, :optin, elem(Timex.parse(data["opt_in"], date_format), 1))

```